packages/app/src/TeamPage.tsx

```diff
@@ -700,6 +700,17 @@ function CreateWebhookForm({
     <form onSubmit={form.handleSubmit(onSubmit)}>
       <Stack mt="sm">
         <Text>Create Webhook</Text>
+        <Radio.Group
+          name="service"
```

Comment:
do we need to register this to the form to persist?

Reply:
Yeah that's a better approach. Its a bit clunky with radio buttons, but I've made the change. It'll persist either way, but using the form hook is cleaner as to not add state outside of the form object itself.